docs/specs/src/hwp_1_intro.md

```diff
@@ -0,0 +1,212 @@
+providing the highest level view of our approach and our starting
+axioms. At the next level of detail we provide an informal system
+description using the metaphor of games. Then we dive into deeper detail
+with a formal system description including the specific context and
+assumptions we come from, the integrity guarantees Holochain offers, and
+a formal state model and discussion of security and safety concerns.
+Finally at the most detailed level we provide an implementation
+specification in Appendix A.
+
+Axioms
+------
+
+We begin by stating our axioms[^1] regarding the nature of coordination:
+
+1.  **Coordination arises from agents** starting from the same
+  ground-rules and acting as soon as each party in any interaction
+  can confirm that each other\'s actions confirm that actions or
+  interactions conform to those ground rules. (Thus, in our frame,
+  coordination looks like alignment heading in the same direction,
+  rather than agreement proceeding in lockstep.)
+
+2.  **Coordination is grammatic**, in that it comes from embodying a
+  geometry that removes uncertainty, and embodying an ability to
+  compose different coordinative subsystems that have different
+  ground-rules.
+
+Axiom 1 arises from the insight that we cannot fight against the
+physical reality of different experiences by different nodes in the
+context of networked interaction nor fight against the real-world
+impossibility of determining global temporal order because of the
+non-existence of simultaneity. Instead, any coordination system must
+align its ontology with the truth that **global state actually does not
+exist**. Thus we start with what does actually exist: local temporal
+state. This local state can be shared with, and validated by, others in
+conformity with pre-defined ground-rules. In so doing we can still
+achieve difficult and complex coordination safely (including in the
+context of problems on the scale of global monetary transactions)
+without the costs and bottlenecks that arise from starting from the
+ontology of a single shared global state. Holochain is an implementation
+of a system using this alternate frame.
+
+Axiom 2 arises from the insight that systems for successful large scale
+coordination demand the property of anti-fragility, that is, they must
+perform better under perturbation[^2]. Coordination happens in the
+context of fundamentally dynamic environments in which the coordinating
+elements are changed by the fact of their coordination. Coordination is
+a co-evolutionary context. We claim by this axiom that what meets the
+challenge of anti-fragility in such contexts is composable sub-systems,
+in which the composition comes out of a grammatics that embodies the
+actual dimensionality of the problem subdomains (i.e. their geometry),
+and by which agents in that context can react powerfully to
+perturbations because the available composability is dimensionally
+aligned.
```

Comment:
Here's that bit on embodying geometry again, with a bit more explanation. I still feel I have to guess at what this means. It is clearly a complex and nuanced thought and the writing could use more explanation, or a reference to some other material.

Is this just saying that the ground rules can be written in an unambiguous way using language (grammar) which fully specifies the problem (sub)domain without room for interpretation? Is it something like, "the map IS the territory"?

Reply:
Well, I'm not actually talking about that determinism.  I'm talking about anti-fragility being met along a gradient by living inside grammatic contexts that actually align with the dimensionality of the space the grammar(s) are about.  English (and all human languages) align grammatically with the context of meaning making.  But it's a gradient.  For example, they fail quite miserably by introducing a grammatic element (the verb to-be) which actually doesn't align with dimensionality of reality of meaning in that that projects on to the outside world something that's actually interior, i.e. "false isness" and hence E-Prime.  So in this regard the gradient of alignment  of the grammar to "reality" is off.  But this is too much to talk about in this intro.

Reply:
This is profound, especially since you're asserting it's at the heart of Holochain design, but I wonder how effective it will be to say this without more explanation. I am left really wanting to know more about what this means, but with no way to find out. Without breaking this down or pointing to some other resource, it may come across as some tantalizing but half-baked thought which it's clearly not. Maybe you could write an appendix on this? I feel like this deserves at least the promise of more detail later, if not the actual deliverance of more detail here and now.

Reply:
Big +1
As Michael said we know there's some real genius behind the Holochain design. But that phrase with "grammatics", "dimensionality", "problem subdomains", "their geometry" and "dimensionally aligned" is a riddle wrapped up in an enigma. It may also be mistaken as a string of lofty words by someone who doesn't want to take the time to understand it.

Reply:
To me a whitepaper is exactly a place where one defines "jargon".  So I guess I need to do that for these terms.  I will add more footnotes.  See above for one on grammatics...

Reply:
I added some footnotes, this may make it worse!

Reply:
Are the footnotes meant to explain what "grammatic embodiment" and "dimensional alignment" mean? The nearby footnotes reference The Players of Ludos and Antifragile, which don't seem to explain these terms.

I really think there needs to be some extra context to support this jargon. I'm not against jargon. There just needs to be a resource for whoever wants to dive in to understand it (including myself), rather than skimming over it.